### PR TITLE
FS-1200: Correct env var spelling for assessment store

### PR DESCRIPTION
**Pull request by rayhan15243**
### Change description

https://digital.dclg.gov.uk/jira/browse/FS-1200

PR to correct the env var spelling for Assessment Store as it was causing the performance test for assessment store to default to Dev when it was running in the pipelines after a deploy to Test.

image

### How to test
N/A

### Screenshots of UI changes (if applicable)
N/A

### CODE REVIEW

File: .github/workflows/pipeline.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
             path: bandit_security.json
             retention-days: 5
         - name: Run performance tests
-          run: python -m locust --users 3 --spawn-rate 1 --run-time 1s --headless
+          run: python -m locust --users 3 --spawn-rate 1 --run-time 15s --headless
```
Comment (tferns) [inline .github/workflows/pipeline.yml:36]: Is this change intentional?

**Review by tferns — APPROVED** [review]
Looks good just one question 👍

**Review by Adam-W1 — APPROVED** [review]

File: .github/workflows/pipeline.yml (hunk 1 above)
Comment (rayhan15243) [reply .github/workflows/pipeline.yml:36]: @tferns Yes was intentional as I had to increase the runtime to accommodate the new microservice for the performance tests.

Comment (rayhan15243) [reply .github/workflows/pipeline.yml:36]: @tferns It's only for this repo, the other repos already have different runtimes which are more than the one here.
